index.html

```diff
@@ -5,44 +5,52 @@
-                <div class="menu-icon"><img src="images/menu.png" alt="follow me on angellist" width="24" height="24" ></div>
+                <div class="nav-div">
+                    <ul class="nav-link">
+                        <li class="nav-link-item"><a href="#">Portfolio</a></li>
+                        <li class="nav-link-item"><a href="#">About</a></li>
+                        <li class="nav-link-item"><a href="#">Contact</a></li>
+                    </ul>
+                </div>
             </nav>
          </header>
             <!--  header section -->
              <section class="first-page">
           <div class="div-header">
 
-           <h1 class="header">I'm Kevin Glad to see you!</h1>  
+           <h1 class="header">I'm Alexandra Glad to see you!</h1>  
          <p class="about-par">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
                
                     <a class="connect-page">LET'S CONNECT</a>
-                
+                <!-- social pages -->
                 <ul class="list">
                     <li class="list-item"><img src="images/tweeter.png" alt="follow me on twitter" width="20" height="20" ></li>
                     <li class="list-item"><img src="images/linkedin.png" alt="follow me on linkedin" width="20" height="20" ></li>
                     <li class="list-item"><img src="images/medium.png" alt="follow me on medium" width="20" height="20" ></li>
                     <li class="list-item"><img src="images/github.png" alt="follow me on github" width="20" height="20" ></li>
                     <li class="list-item"><img src="images/angelist.png" alt="follow me on angellist" width="20" height="20" ></li>
                 </ul>
-                </div>
+            </div>
         </section>  
 <!-- work section -->
             <section class="work-section">
-               <div class="card">
-            <div class="section-img"><img src="images/snapshoot1.png"></div>
+
+                <!-- ....card1 div.... -->
+               <div class="card card1">
+            <div class="section-img"><img src="images/snapshoot5.png"></div>
             <div class="card-info">
             <h2 class="work-header">Tonic</h2>
             <ul class="descr-list">
                 <li class="descr-list-item1">CANOPY</li>
-                <li class="descr-list-item"><img class="counter-img" src="images/Counter.png"> End Dev</li>
+                <li class="descr-list-item"><img class="counter-img" src="images/Counter.png">Back End Dev</li>
                 <li class="descr-list-item"><img class="counter-img"src="images/Counter.png">2015</li>
             </ul>
             <p class="lang-descr">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
```

Comment:
- [ ] [REQUIRED] Good job so far! Please give the correct `space` identifications value(`padding..e.t.c`) to the `text` and `list items` in the work section, because their location is not like [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=Uy7TAUKVJbGSY3Ol-0), also when `hovering` each project card, give the `border color` like in [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=Uy7TAUKVJbGSY3Ol-0) 🙏🏼 

![image](https://user-images.githubusercontent.com/63915024/213463496-5c7cda90-91a0-42d2-ac14-7a8ab3896bc2.png)